tests/Ethereum/TWCoinTypeTests.cpp

```diff
@@ -15,10 +15,10 @@
 
 TEST(TWEthereumCoinType, TWCoinType) {
     auto symbol = WRAPS(TWCoinTypeConfigurationGetSymbol(TWCoinTypeEthereum));
-    auto txId = TWStringCreateWithUTF8Bytes("0x9edaf0f7d9c6629c31bbf0471fc07d696c73b566b93783f7e25d8d5d2b62fa4f");
-    auto txUrl = WRAPS(TWCoinTypeConfigurationGetTransactionURL(TWCoinTypeEthereum, txId));
-    auto accId = TWStringCreateWithUTF8Bytes("0x5bb497e8d9fe26e92dd1be01e32076c8e024d167");
-    auto accUrl = WRAPS(TWCoinTypeConfigurationGetAccountURL(TWCoinTypeEthereum, accId));
+    auto txId = WRAPS(TWStringCreateWithUTF8Bytes("0x9edaf0f7d9c6629c31bbf0471fc07d696c73b566b93783f7e25d8d5d2b62fa4f"));
```

Comment:
don't forget to fix `TWCoinTypeTests.cpp.erb`

Reply:
Good thinking :)  I did, but for easier review I included only test files in this PR.  There are 2-3 non-test changes, those I will include in the ASAN PR.